model.py

```diff
@@ -127,12 +131,13 @@ def _make_thumbs(self):
         self.scaled_joints = self._initjoints()
 
         for key in self.scaled_sticks:
-            (angle,len) = self.scaled_sticks[key]
+            (angle, len) = self.scaled_sticks[key]
             newlen = int(len * .2)
-            self.scaled_sticks[key] = (angle,newlen)
+            self.scaled_sticks[key] = (angle, newlen)
 
         self._setjoints(self.scaled_joints, self.scaled_sticks,
-                (self.x, KEYFRAMEHEIGHT/2))
+                        (self.x, theme.KEYFRAMEHEIGHT / 2))
```

Comment:
Maybe use floor division here as you're dividing by an int but test to make sure the dimensions are as they should be so if `theme.KEYFRAMEHEIGHT` is  a float then no need to change it.